contrib/gazebo/sawyer/pick_and_place.py

```diff
@@ -0,0 +1,33 @@
+"""
+Pick-and-place task for the sawyer robot
+"""
+
+from contrib.gazebo import gazebo_sawyer_env
+from rllab.core.serializable import Serializable
```

Comment:
PEP8 import order

Reply:
I went through the imports section in PEP8 and I found I should group imports into three groups(standard packages, third party packages, local packages/sources). Do you mean the two imports here belong to different groups. I thought they are both local packages. Let me know if I was wrong!

Reply:
We are going to extend that logic, since rllab has its own hierarchy of import paths.

For example (obviously comments would not appear in real code):
```python
# python standard library
import os

# third-party packages
import tensorflow as tf
import numpy as np

# rllab top-level tree
from rllab.envs.mujoco.swimmer_env import SwimmerEnv

# rllab sandboxes
from sandbox.rocky.tf.policies.gaussian_mlp_policy import GaussianMLPPolicy

# project-local code (e.g. contrib in this case)
from contrib.gazebo import gazebo_sawyer_env
```